### PR TITLE
msg/mds: print more debug logs from protocol layer

### PR DESCRIPTION
**Pull request by lxbsz**
The bug is for many time the MDS sent out the `beacon` msg to monitors,
and from the mds and msgr logs we can see that the `beacon` msg was
really sent out, more exactly is the msg had been enqueued to the protocol
sender queue. But we don't know whether this msg had been sent out or
stuck in the route, and the monitors received it after minutes.

We need to know whether it's a bug in protocol code and need more debug
logs from it.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [x] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: qa/suites/fs/functional/tasks/snap-schedule.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ overrides:
     conf:
       mgr:
         debug mgr: 20
-        debug ms: 1
+        debug ms: 2
```
Comment (batrick) [inline qa/suites/fs/functional/tasks/snap-schedule.yaml:6]: If we are changing `qa/cephfs/conf` then why do we need these changes?

**Review by batrick — CHANGES_REQUESTED** [review]
mon.yaml too?

Why mgr.yaml?

Please change `URL: ` in commit message to `Related: ` or similar.

**Comment by lxbsz** [comment]
> mon.yaml too?
> 
> Why mgr.yaml?
> 
> Please change `URL: ` in commit message to `Related: ` or similar.

Actually this could happen in other components too in theory. For now let's do it in mds only and I will remove the change for mon and mgr.

File: qa/suites/fs/functional/tasks/snap-schedule.yaml (hunk 1 above)
Comment (lxbsz) [reply qa/suites/fs/functional/tasks/snap-schedule.yaml:6]: I thought the `conf` will be overrode by this ?

Comment (batrick) [reply qa/suites/fs/functional/tasks/snap-schedule.yaml]: no, they are merged together

Comment (lxbsz) [reply qa/suites/fs/functional/tasks/snap-schedule.yaml]: Okay.

**Review by batrick — APPROVED** [review]

**Comment by batrick** [comment]
This PR is under test in [https://tracker.ceph.com/issues/66822](https://tracker.ceph.com/issues/66822).

**Comment by batrick** [comment]
```
./7786768/remote/smithi193/log/8c7125b8-39d1-11ef-bcab-c7b262605968/ceph-mds.l.log.gz:2024-07-04T06:58:29.805+0000 7f4fe61f6640  2 --2- [v2:172.21.15.193:6832/3835154050,v1:172.21.15.193:6833/3835154050] >> [v2:172.21.15.193:3300/0,v1:172.21.15.193:6789/0] conn(0x561666f3b180 0x561667b1a000 secure :-1 s=READY pgs=52 cs=0 l=1 rev1=1 crypto rx=0x561662d639b0 tx=0x5616670b7140 comp rx=0 tx=0).write_message sending message m=0x561666e01800 seq=1 mon_getmap magic: 0
./7786768/remote/smithi193/log/8c7125b8-39d1-11ef-bcab-c7b262605968/ceph-mds.l.log.gz:2024-07-04T06:58:29.805+0000 7f4fe61f6640  2 --2- [v2:172.21.15.193:6832/3835154050,v1:172.21.15.193:6833/3835154050] >> [v2:172.21.15.193:3300/0,v1:172.21.15.193:6789/0] conn(0x561666f3b180 0x561667b1a000 secure :-1 s=READY pgs=52 cs=0 l=1 rev1=1 crypto rx=0x561662d639b0 tx=0x5616670b7140 comp rx=0 tx=0).write_message sending message m=0x561667b323c0 seq=2 mon_subscribe({config=0+,monmap=0+})
./7786768/remote/smithi193/log/8c7125b8-39d1-11ef-bcab-c7b262605968/ceph-mds.l.log.gz:2024-07-04T06:58:29.805+0000 7f4fe61f6640  2 --2- [v2:172.21.15.193:6832/3835154050,v1:172.21.15.193:6833/3835154050] >> [v2:172.21.15.193:3300/0,v1:172.21.15.193:6789/0] conn(0x561666f3b180 0x561667b1a000 secure :-1 s=READY pgs=52 cs=0 l=1 rev1=1 crypto rx=0x561662d639b0 tx=0x5616670b7140 comp rx=0 tx=0).write_message sending message m=0x561666deac40 seq=3 auth(proto 2 2 bytes epoch 0)
./7786768/remote/smithi193/log/8c7125b8-39d1-11ef-bcab-c7b262605968/ceph-mds.l.log.gz:2024-07-04T06:58:29.805+0000 7f4fe61f6640  2 --2- [v2:172.21.15.193:6832/3835154050,v1:172.21.15.193:6833/3835154050] >> [v2:172.21.15.193:3300/0,v1:172.21.15.193:6789/0] conn(0x561666f3b180 0x561667b1a000 secure :-1 s=READY pgs=52 cs=0 l=1 rev1=1 crypto rx=0x561662d639b0 tx=0x5616670b7140 comp rx=0 tx=0).write_message sending message m=0x561667b325a0 seq=4 mon_subscribe({mdsmap=0+})
./7786768/remote/smithi193/log/8c7125b8-39d1-11ef-bcab-c7b262605968/ceph-mds.l.log.gz:2024-07-04T06:58:29.806+0000 7f4fe61f6640  2 --2- [v2:172.21.15.193:6832/3835154050,v1:172.21.15.193:6833/3835154050] >> [v2:172.21.15.193:3300/0,v1:172.21.15.193:6789/0] conn(0x561666f3b180 0x561667b1a000 secure :-1 s=READY pgs=52 cs=0 l=1 rev1=1 crypto rx=0x561662d639b0 tx=0x5616670b7140 comp rx=0 tx=0).write_message sending message m=0x561666df2600 seq=5 mdsbeacon(24550/l up:boot seq=1 v0)
./7786768/remote/smithi193/log/8c7125b8-39d1-11ef-bcab-c7b262605968/ceph-mds.l.log.gz:2024-07-04T06:58:29.808+0000 7f4fe61f6640  2 --2- [v2:172.21.15.193:6832/3835154050,v1:172.21.15.193:6833/3835154050] >> [v2:172.21.15.193:3300/0,v1:172.21.15.193:6789/0] conn(0x561666f3b180 0x561667b1a000 secure :-1 s=READY pgs=52 cs=0 l=1 rev1=1 crypto rx=0x561662d639b0 tx=0x5616670b7140 comp rx=0 tx=0).write_message sending message m=0x561667b32d20 seq=6 mon_subscribe({mgrmap=0+})
...
```

works as advertised.

**Review by pereman2 — COMMENTED** [review]
https://tracker.ceph.com/projects/rados/wiki/MAIN#httpstrackercephcomissues67028 rados approved

**Pull request merged**